docs/preset-env.md

```diff
@@ -82,10 +82,12 @@ For more information on setting options for a preset, refer to the [preset optio
 
 ### `targets`
 
-`string | Array<string> | { [string]: string }`, defaults to the top-level `targets` option if no browserslist-related option is specified in `@babel/preset-env`'s docs, otherwise to `{}`.
+`string | Array<string> | { [string]: string }`, defaults to the top-level `targets` option if no [browserslist config sources](https://github.com/ai/browserslist#queries) are specified, otherwise to `{}`.
```

Comment:
"browserslist-related options" are actually more than the config sources. Namely they are `configPath`, `browserslistEnv` and `ignoreBrowserslistConfig`. So I prefer to keep previous wordings, or we can clarify what are these options.

Reply:
```suggestion
`string | Array<string> | { [string]: string }`, defaults to the top-level [`targets` option](options.md#targets) if [`configPath`](#configpath), [`browserslistEnv`](#browserslistenv) and [`ignoreBrowserslistConfig`](#ignorebrowserslistconfig) are not specified, otherwise to `{}`.
```

The browserslist config sources can be found in `options.md#targets`, as we have put below:

```md
> For usage, refer to the [`targets` option](options.md#targets) documentation.
```

For record there was a link to the browserslist query before we are promoting the top level `targets` option, which can benefit not only `preset-env` but also other plugins. After that most of the `targets` docs are moved to `options.md#targets`. 

I think this section should only cover the behaviour different to the top-level `targets`, like the note you have put about the `esmodules` intersection. Users are supposed to refer to `options.md#targets` about how `targets` should be configured. Otherwise we will have almost identical two `targets` configuration guides.

Reply:
I trust you understand the subtleties more than me. I've left this PR open to modifications so you can tune this appropriately.